### PR TITLE
Feature/entry data version

### PR DESCRIPTION
**Pull request by pflueras**
Implemented entryVersion for remaining actors.
Related to https://github.com/vlingo/vlingo-symbio-jdbc/issues/37 issue.

### CODE REVIEW

File: src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCDispatcherControlDelegate.java [hunk 1]
```diff
@@ -146,11 +146,11 @@ private void fail() {
         final String entryType = resultSet.getString(3);
         final int eventTypeVersion = resultSet.getInt(4);
         final String entryMetadata = resultSet.getString(5);
-        final int entryVerion = resultSet.getInt(6); // from E_STREAM_VERSION
+        final int entryVersion = resultSet.getInt(6); // from E_STREAM_VERSION
```
Comment (VaughnVernon) [inline src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCDispatcherControlDelegate.java]: @pflueras I don't understand what changed here.

File: src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCJournalReaderActor.java [hunk 2]
```diff
@@ -187,12 +187,12 @@ private TextEntry entryFromResultSet(final ResultSet resultSet) throws SQLExcept
         final String entryType = resultSet.getString(3);
         final int eventTypeVersion = resultSet.getInt(4);
         final String entryMetadata = resultSet.getString(5);
-        final int entryVerion = resultSet.getInt(6); // from E_STREAM_VERSION
+        final int entryVersion = resultSet.getInt(6); // from E_STREAM_VERSION
```
Comment (VaughnVernon) [inline src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCJournalReaderActor.java]: @pflueras I don't understand what changed here.

File: src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCDispatcherControlDelegate.java [hunk 3]
```diff
@@ -146,11 +146,11 @@ private void fail() {
...
 
         final Class<?> classOfEvent = Class.forName(entryType);
 
         final Metadata metadata = JsonSerialization.deserialized(entryMetadata, Metadata.class);
-        return new BaseEntry.TextEntry(id, classOfEvent, eventTypeVersion, entryData, entryVerion, metadata);
+        return new BaseEntry.TextEntry(id, classOfEvent, eventTypeVersion, entryData, entryVersion, metadata);
```
Comment (VaughnVernon) [inline src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCDispatcherControlDelegate.java]: @pflueras I don't understand what changed here.

File: src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCJournalReaderActor.java [hunk 4]
```diff
@@ -187,12 +187,12 @@ private TextEntry entryFromResultSet(final ResultSet resultSet) throws SQLExcept
...
 
         final Class<?> classOfEvent = Class.forName(entryType);
         final Metadata eventMetadataDeserialized = gson.fromJson(entryMetadata, Metadata.class);
 
-        return new BaseEntry.TextEntry(String.valueOf(id), classOfEvent, eventTypeVersion, entryData, entryVerion, eventMetadataDeserialized);
+        return new BaseEntry.TextEntry(String.valueOf(id), classOfEvent, eventTypeVersion, entryData, entryVersion, eventMetadataDeserialized);
```
Comment (VaughnVernon) [inline src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCJournalReaderActor.java]: @pflueras I don't understand what changed here.

**Comment by VaughnVernon** [comment]
@pflueras I just made a few comments/observations. Please merge when ready.

File: src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCJournalReaderActor.java (hunk 2 above)
Comment (pflueras) [reply src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCJournalReaderActor.java]: It was a typo: entryVerion. Missing 's'.

File: src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCDispatcherControlDelegate.java (hunk 1 above)
Comment (pflueras) [reply src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCDispatcherControlDelegate.java]: It was a typo: entryVerion. Missing 's'.

File: src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCDispatcherControlDelegate.java (hunk 3 above)
Comment (pflueras) [reply src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCDispatcherControlDelegate.java]: It was a typo: entryVerion. Missing 's'.

File: src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCJournalReaderActor.java (hunk 4 above)
Comment (pflueras) [reply src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCJournalReaderActor.java]: It was a typo: entryVerion. Missing 's'.

**Comment by pflueras** [comment]
Unit tests are extended to cover various cases of `entryVersion`.

**Comment by VaughnVernon** [comment]
@pflueras 👍 💯 🥇"entryVerion" -- my tired eyes didn't see it. Thanks for taking care of that.

**Pull request merged**